Makefile

```diff
@@ -1,17 +1,21 @@
 default: build
 
 CMD_IMPORT ?= github.com/heptio/authenticator/cmd/heptio-authenticator-aws
+GITHUB_REPO ?= github.com/heptio/authenticator
 REPO ?= gcr.io/heptio-images/authenticator
 VERSION ?= v0.1.0
 
-.PHONY: build push build-container
+.PHONY: test build push build-container
 
-build: build-container heptio-authenticator-aws-osx
+build: test build-container heptio-authenticator-aws-osx
```

Comment:
`test` could probably just be a dependency to `heptio-authenticator-aws-osx`.

Reply:
If it is only a target for osx then it will not run when build-container is run explicitly.